### PR TITLE
Do not fail test on HCO removal timeout.

### PR DESCRIPTION
**Pull request by mrnold**
Address one test failure seen in #1253, where HCO removal timed out during cleanup. This pull request avoids failing the virtualization tests when HCO removal times out, and increases the timeout by one minute on the off chance that it helps.

### CODE REVIEW

**Comment by mateusoliveira43** [comment]
How many times have the clean up failed? if its not something happening more than once a week, I would not merge this one yet

**Review by kaovilai — APPROVED** [review]

**Comment by mrnold** [comment]
> How many times have the clean up failed? if its not something happening more than once a week, I would not merge this one yet

The change that caused it has only been merged for two days so far, so I would guess it's going to happen more than once a week. We can hold it if you want to see for sure.

**Comment by mpryc** [comment]
Another failure caused by this: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_oadp-operator/1253/pull-ci-openshift-oadp-operator-master-4.13-e2e-test-aws/1753060465228910592

**Pull request merged**
